cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/LegacyCommonGridModelExportTest.java

```diff
@@ -0,0 +1,145 @@
+package com.powsybl.cgmes.conversion.test.export;
+
+import com.powsybl.cgmes.conformity.CgmesConformity1ModifiedCatalog;
+import com.powsybl.cgmes.conversion.CgmesExport;
+import com.powsybl.cgmes.extensions.CgmesMetadataModels;
+import com.powsybl.cgmes.extensions.CgmesMetadataModelsAdder;
+import com.powsybl.cgmes.model.CgmesMetadataModel;
+import com.powsybl.cgmes.model.CgmesSubset;
+import com.powsybl.commons.datasource.ReadOnlyDataSource;
+import com.powsybl.commons.test.AbstractSerDeTest;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.util.Identifiables;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.powsybl.cgmes.conversion.Conversion.CGMES_PREFIX_ALIAS_PROPERTIES;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
+ */
+class LegacyCommonGridModelExportTest extends AbstractSerDeTest {
+
+    @Test
+    void testExportCgmSvDependenciesOnNetworkProperties() throws IOException {
+        ReadOnlyDataSource ds = CgmesConformity1ModifiedCatalog.microGridBaseCaseAssembledSvWithMas().dataSource();
+        Network network = Network.read(ds);
+
+        // This is the legacy way of preparing dependencies for SV externally,
+        // used by projects in the FARAO community
+        network.setProperty(Identifiables.getUniqueId(CGMES_PREFIX_ALIAS_PROPERTIES + "SSH_ID", network::hasProperty), "ssh-updated-dep1");
+        network.setProperty(Identifiables.getUniqueId(CGMES_PREFIX_ALIAS_PROPERTIES + "SSH_ID", network::hasProperty), "ssh-updated-dep2");
+        network.setProperty(Identifiables.getUniqueId(CGMES_PREFIX_ALIAS_PROPERTIES + "SSH_ID", network::hasProperty), "ssh-updated-dep3");
+        network.setProperty(Identifiables.getUniqueId(CGMES_PREFIX_ALIAS_PROPERTIES + "TP_ID", network::hasProperty), "tp-initial-dep1");
+        network.setProperty(Identifiables.getUniqueId(CGMES_PREFIX_ALIAS_PROPERTIES + "TP_ID", network::hasProperty), "tp-initial-dep2");
+        network.setProperty(Identifiables.getUniqueId(CGMES_PREFIX_ALIAS_PROPERTIES + "TP_ID", network::hasProperty), "tp-initial-dep3");
+
+        Properties exportParams = new Properties();
+        exportParams.put(CgmesExport.PROFILES, "SV");
+        String exportBasename = "tmp-micro-bc-from-CGM";
+        network.write("CGMES", exportParams, tmpDir.resolve(exportBasename));
+
+        Set<String> deps = findAll(REGEX_DEPENDENT_ON, Files.readString(tmpDir.resolve(exportBasename + "_SV.xml")));
+        System.out.println("dependencies : " + Arrays.toString(deps.toArray()));
+        assertEquals(Set.of("ssh-updated-dep1", "ssh-updated-dep2", "ssh-updated-dep3", "tp-initial-dep1", "tp-initial-dep2", "tp-initial-dep3"), deps);
+    }
+
+    @Test
+    void testExportCgmSvDependenciesOnMetadataModelsExtension() throws IOException {
```

Comment:
This way never relies on properties, right?

Reply:
Yes, this test uses only metadata models. No properties are used or required.